<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>780</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>0</y>
         <width>200</width>
         <height>60</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI Black</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:rgb(255, 255, 255);
background-color: rgba(255, 255, 255, 0);</string>
       </property>
       <property name="text">
        <string>SUPER BENMARK</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>25</x>
         <y>5</y>
         <width>200</width>
         <height>60</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI Black</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #aa00ff;</string>
       </property>
       <property name="text">
        <string>SUPER BENMARK</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>50</y>
         <width>750</width>
         <height>750</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>circleloop3.gif</pixmap>
       </property>
      </widget>
      <widget class="QPushButton" name="exitButton">
       <property name="geometry">
        <rect>
         <x>1020</x>
         <y>15</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI Light</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">
QPushButton{color: rgba(255, 255, 255, 100);}
QPushButton:hover{
	
	color: rgb(39, 47, 226);
}</string>
       </property>
       <property name="text">
        <string>X</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>650</y>
         <width>131</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
         <underline>false</underline>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0.414773, stop:0 #aa00ff, stop:1 #ff73d3);\n&quot;&quot;padding: 10px;
padding: 10px;
text-align: center;
border-style: none;
}
QPushButton:hover{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0.414773, stop:0 #ff73d3, stop:1 #aa00ff );
	margin-left:5px;
	margin-top:5px;
	font-size: 14px;
}</string>
       </property>
       <property name="text">
        <string>BACK</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>105</y>
         <width>350</width>
         <height>60</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>20</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">
color: #aa00ff;</string>
       </property>
       <property name="text">
        <string>System Information</string>
       </property>
      </widget>
      <widget class="QFrame" name="frame_2">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>110</y>
         <width>641</width>
         <height>461</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QWidget" name="gridLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>110</y>
          <width>561</width>
          <height>361</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_2" columnstretch="1,2">
         <item row="4" column="0">
          <widget class="QLabel" name="label_23">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>processor</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_18">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>Architecture</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_17">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>Host name</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="host_name_2">
           <property name="styleSheet">
            <string notr="true">color: rgb(218, 218, 218);
background-color: rgba(255, 255, 255, 0);</string>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_22">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>mac - address</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_24">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>RAM</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLabel" name="address_2">
           <property name="styleSheet">
            <string notr="true">color: rgb(218, 218, 218);
background-color: rgba(255, 255, 255, 0);</string>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLabel" name="ram_2">
           <property name="styleSheet">
            <string notr="true">color: rgb(218, 218, 218);
background-color: rgba(255, 255, 255, 0);</string>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="arcitecture">
           <property name="styleSheet">
            <string notr="true">color: rgb(218, 218, 218);
background-color: rgba(255, 255, 255, 0);</string>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="platform_2">
           <property name="styleSheet">
            <string notr="true">color: rgb(218, 218, 218);
background-color: rgba(255, 255, 255, 0);</string>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLabel" name="processor_2">
           <property name="styleSheet">
            <string notr="true">color: rgb(218, 218, 218);
background-color: rgba(255, 255, 255, 0);</string>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_16">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>Platform</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_25">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>Memmory</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QLabel" name="ram_3">
           <property name="styleSheet">
            <string notr="true">color: rgb(218, 218, 218);
background-color: rgba(255, 255, 255, 0);</string>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <zorder>label_3</zorder>
      <zorder>label_2</zorder>
      <zorder>label</zorder>
      <zorder>exitButton</zorder>
      <zorder>pushButton_2</zorder>
      <zorder>label_15</zorder>
      <zorder>frame_2</zorder>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
